README.md

```diff
@@ -1,12 +1,29 @@
 # NuRadioMC and NuRadioReco
-NuRadioMC: A Monte Carlo simulation package for radio neutrino detectors.
+**NuRadioMC**: A Monte Carlo simulation package for radio neutrino detectors.
 
-NuRadioReco: A reconstruction and detector simulation framework for radio detectors of high-energy neutrinos and cosmic-rays. (NuRadioReco is independent of NuRadioMC
-and used by NuRadioMC for the detector and trigger simulation. To simplify the development and continuous integration testing, both codes are developed 
+**NuRadioReco**: A reconstruction and detector simulation framework for radio detectors of high-energy neutrinos and cosmic-rays. (NuRadioReco is independent of NuRadioMC
+and used by NuRadioMC for the detector and trigger simulation. To simplify the development and continuous integration testing, both codes are developed
 in the same github repository.)
 
-The documentation can be found at https://nu-radio.github.io/NuRadioMC/main.html
+## Documentation and Installation
 
+The documentation including detailed installation instructions can be found at https://nu-radio.github.io/NuRadioMC/main.html
+
+**TL;DR:**
+
+Developers should clone the git repository and run:
+```
+pip install -e .[dev]  # zsh user might have to wrap the argument in quotes: -e '.[dev]'
+pre-commit install
+```
+Additional optional dependencies are e.g. `-e [rno-g,proposal,galacticnoise,cr_interpolator]`. To install all optional dependencies use `-e [all]`
```

Comment:
`-e [all]` doesn't work - either specify to just add `[all]` or copy `pip install -e .[all,dev]` in its entirety.